internal/sources/cloudsqlmssql/cloud_sql_mssql.go

```diff
@@ -49,16 +49,15 @@ func newConfig(ctx context.Context, name string, decoder *yaml.Decoder) (sources
 
 type Config struct {
 	// Cloud SQL MSSQL configs
-	Name      string         `yaml:"name" validate:"required"`
-	Kind      string         `yaml:"kind" validate:"required"`
-	Project   string         `yaml:"project" validate:"required"`
-	Region    string         `yaml:"region" validate:"required"`
-	Instance  string         `yaml:"instance" validate:"required"`
-	IPAddress string         `yaml:"ipAddress" validate:"required"`
```

Comment:
Should we keep the field listed here but deprecate it (similar to AuthSources) so it's not a breaking change?